### ISSUE TITLE
Events detail review

### ISSUE THREAD

**Issue by jasloe** [aarome/aarome#167]
Same deal as people. New template, new stylesheet. 

https://dev.aarome.org/events/calendar/2019-mckim-medal-gala
https://dev.aarome.org/it/eventi/calendario/2019-mckim-medal-gala

https://dev.aarome.org/events/calendar/virginia-virilli
https://dev.aarome.org/it/eventi/calendario/virginia-virilli

https://dev.aarome.org/events/calendar/2019-arthur-janet-c-ross-rome-prize-ceremony
https://dev.aarome.org/it/eventi/calendario/2019-arthur-janet-c-ross-rome-prize-ceremony

Don't think we ever resolved where to stick the event series. 

My thought is that it should be in the breadcrumb, such as:

Eventi / Calendario / Shoptalks
Events / Calendar / McKim Medal Gala 
Events / Calendar / Conversations/Conversazioni 

Will put a few minutes into that now just to see what it'll take.

News features tomorrow.

**Comment by jasloe** [comment]
For example: 

https://dev.aarome.org/events/calendar/fellow-shoptalks/franco-baldasso-dylan-francareta
https://dev.aarome.org/it/eventi/calendario/fellow-shoptalks/franco-baldasso-dylan-francareta

If an event has no series assigned, it'll bail out with no series in the breadcrumb or URL:

https://dev.aarome.org/events/calendar/stephen-greenblatt-lucretius-survival-dangerous-ideas
https://dev.aarome.org/it/eventi/calendario/stephen-greenblatt-lucretius-survival-dangerous-ideas

Note that the event series component of the breadcrumb would take the user to a list of events carrying that term.

**Comment by cchristopherhowardd** [comment]
What a fantastic idea!

**Comment by cchristopherhowardd** [comment]
Again, a great idea. The functionality may need finessing. For instance, on the Franco Baldasso event page, clicking Fellow Shoptalks in the breadcrumb takes you to https://dev.aarome.org/events/calendar/fellow-shoptalks. Is this page going to be a mirror to https://dev.aarome.org/events/series and choosing Fellow Shoptalks from the drop-down menu? Image of that path below.

<img width="1174" alt="Screen Shot 2019-03-22 at 11 09 05 AM" src="https://user-images.githubusercontent.com/38765846/54832766-2e2b3d80-4c93-11e9-891d-4782061cc8b6.png">

**Comment by jasloe** [comment]
I think so? Let me think about this a little more. 

Question: 

All events have event an type term but not all events have an even series term, correct?

**Comment by jasloe** [comment]
> What a fantastic idea!

I get them once in a very long while.

**Comment by cchristopherhowardd** [comment]
Yes, all events have a type term, but not all events are part of a series. However, some events are part of two series, so we need an option to include two series for a single event. I expressed this before but think it was misunderstood. You created an option for selecting two type terms for an event, not two series types.

I suppose the two term types can stay--I plan to select only one for events as much as possible--but occasionally an event will be a film screening followed by a conversation. I don't want to open term types any further, because that leaves too much room for AAR staff members who aren't me to select any term types that may apply to an event, under the impression that being as inclusive as possible is a good thing. Categories are meant to categorize.

<img width="960" alt="Screen Shot 2019-03-27 at 9 45 14 AM" src="https://user-images.githubusercontent.com/38765846/55081369-1e866d00-5076-11e9-9d30-dcdf1301cf9a.png">

**Comment by jasloe** [comment]
Two recently created nodes have more than one event type:

https://dev.aarome.org/node/4662/edit
https://dev.aarome.org/node/4670/edit

One of the terms needs to be deleted in order to change the configuration to restrict event type selection to one term. 

In the meantime, I've restricted the event type selection to two terms. 

The Event series field is now restricted to two terms.

**Comment by cchristopherhowardd** [comment]
I deleted one event type term from each page.
